packages/@expo/cli/src/start/server/middleware/CorsMiddleware.ts

```diff
@@ -54,6 +54,9 @@ export function createCorsMiddleware(exp: ExpoConfig) {
         );
         return;
       } else if (!isLocalhost && isAllowedHost) {
+        // Skipped for localhost since we shouldn't allow escalating cross-origin requests
+        // across separate sites running on localhost. If the `Origin` is set to a localhost
+        // hostname, we're not making a cross-origin request anyway
```

Comment:
Sorry for nitpicking here, I think it'd be an improvement to say something to the tune of "If the request origin has a "localhost" hostname, we disallow cross-origin requests in order to allow requests only from this dev server's webpages."